web/src/documents.e2e.ts

```diff
@@ -55,3 +64,46 @@ test('basic', async () => {
   // Close the document
   await close(client, document.id)
 })
+
+test('create', async () => {
+  let document = await create(client)
+  let json = await dump(client, document.id, 'json')
+  expect(json).toMatch(/^{"type":"Article"/)
+
+  document = await create(client, creativeWork({ content: ['Beep!'] }))
+  let md = await dump(client, document.id, 'md')
+  expect(md).toMatch(/^Beep!/)
+
+  document = await create(client, 'Boop!', 'md')
+  json = await dump(client, document.id, 'json')
+  expect(json).toMatch(/"content":\["Boop!"\]/)
+})
+
+test('load', async () => {
+  let document = await create(client)
+
+  let ok = await load(client, document.id, article({ content: [] }))
+  expect(ok).toBeTruthy()
+  let json = await dump(client, document.id, 'json')
+  expect(json).toMatch(/^{"type":"Article"/)
+
+  ok = await load(client, document.id, 'Hello *world*!', 'md')
+  expect(ok).toBeTruthy()
+  json = await dump(client, document.id, 'html')
+  expect(json).toMatch(
+    '<em itemtype="http://schema.stenci.la/Emphasis" itemscope><span>world</span>'
+  )
+})
+
+test('dump', async () => {
+  let document = await open(client, 'fixtures/nodes/creative-work.json')
+
+  let json = await dump(client, document.id, 'json')
+  expect(json).toMatch(/^{"type":"CreativeWork"/)
+
+  let md = await dump(client, document.id, 'md')
+  expect(md).toMatch(/^A fixture that is a creative work/)
+
+  let rpng = await dump(client, document.id, 'rpng', 'cc-1')
+  expect(rpng).toMatch(/^data:image\/png/)
```

Comment:
@alex-ketch This is how to get a base64 encoded RPNG for a node. See other tests above for exampes of dumping the whole document and using other formats.